src/apis/impl/HealthcareProfessionalApiImpl.ts

```diff
@@ -10,20 +10,31 @@ import {
   IccHcpartyXApi,
   IccHelementXApi,
   IccPatientXApi,
-  IccUserXApi
+  IccUserXApi, MaintenanceTask
 } from "@icure/api";
 import {HealthcareProfessionalMapper} from "../../mappers/healthcareProfessional";
 import {PaginatedListMapper} from "../../mappers/paginatedList";
 import {FilterMapper} from "../../mappers/filter";
 import {firstOrNull} from "../../utils/functionalUtils";
+import {IccMaintenanceTaskXApi} from "@icure/api/icc-x-api/icc-maintenance-task-x-api";
 
 export class HealthcareProfessionalApiImpl implements HealthcareProfessionalApi {
   private readonly userApi: IccUserXApi;
   private readonly hcpApi: IccHcpartyXApi;
+  private readonly maintenanceApi: IccMaintenanceTaskXApi;
 
-  constructor(api: { cryptoApi: IccCryptoXApi; userApi: IccUserXApi; patientApi: IccPatientXApi; contactApi: IccContactXApi; documentApi: IccDocumentXApi; healthcarePartyApi: IccHcpartyXApi, healthcareElementApi: IccHelementXApi}) {
+  constructor(api: { cryptoApi: IccCryptoXApi; userApi: IccUserXApi; patientApi: IccPatientXApi; contactApi: IccContactXApi; documentApi: IccDocumentXApi; healthcarePartyApi: IccHcpartyXApi, healthcareElementApi: IccHelementXApi, maintenanceTaskApi: IccMaintenanceTaskXApi}) {
     this.userApi = api.userApi;
     this.hcpApi = api.healthcarePartyApi;
+    this.maintenanceApi = api.maintenanceTaskApi;
+  }
+
+  async createMaintenanceTask(hcpId: string | undefined, userId: string | undefined): Promise<MaintenanceTask> {
```

Comment:
This one should not exist ffrom my point of view : It can be removed and replaced by NotificationService of the MedTech SDK